docs/source/type_info.rst

```diff
@@ -19,16 +19,16 @@ A :class:`torch.finfo` is an object that represents the numerical properties of
 
 A :class:`torch.finfo` provides the following attributes:
 
-=========   =====   ========================================
+==========  =====   ========================================
 Name        Type    Description
-=========   =====   ========================================
+==========  =====   ========================================
 bits        int     The number of bits occupied by the type.
 eps         float   The smallest representable number such that ``1.0 + eps != 1.0``.
 max         float   The largest representable number.
 min         float   The smallest representable number (typically ``-max``).
 tiny        float   The smallest positive representable number.
 resolution  float   The approximate decimal resolution of this type, i.e., ``10**-precision``.
-=========   =====   ========================================
+==========  =====   ========================================
```

Comment:
The addition of the "resolution" row broke formatting. This happened sometime in the past week, making the case stronger for not allowing warnings to pass, gh-38011